third_party/yapf_format_diff/yapf_format_diff.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
yapf already includes "formatter" in the name. Let's stick with $TOOL + diff for the name (the formatter is called clang-format in the other case). Also, can it be yapf-diff.py to match clang-format-diff.py?

Reply:
Done.